<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuUsr</class>
 <widget class="QMainWindow" name="MenuUsr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MenuUsr</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Bienvenido:</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignRight">
       <widget class="QPushButton" name="pushButtonCerrarSesion">
        <property name="text">
         <string>Cerrar Sesión</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageBuscar">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lineEditCorreo">
            <property name="placeholderText">
             <string>Ingrese correo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonBuscar">
            <property name="text">
             <string>Buscar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxDatosUsuario">
          <property name="title">
           <string>Datos de Usuario</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="labelNombres">
             <property name="text">
              <string>Nombres:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelApellidos">
             <property name="text">
              <string>Apellidos:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelCorreo">
             <property name="text">
              <string>Correo:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelFechaNacimiento">
             <property name="text">
              <string>Fecha de Nacimiento:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pagePublicaciones">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QGroupBox" name="groupBoxFecha">
            <property name="title">
             <string>Fecha</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QComboBox" name="comboBoxFecha">
               <item>
                <property name="text">
                 <string>Todas</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonAplicarFecha">
               <property name="text">
                <string>Aplicar</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBoxRecorridoLimitado">
            <property name="title">
             <string>Recorrido Limitado</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="labelOrden">
                 <property name="text">
                  <string>Orden:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBoxOrden">
                 <item>
                  <property name="text">
                   <string>PreOrden</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>PostOrden</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>InOrden</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="labelCantidad">
                 <property name="text">
                  <string>Cantidad:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEditCantidad"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonAplicarRecorrido">
               <property name="text">
                <string>Aplicar</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QPushButton" name="pushButtonCrearPublicacion">
            <property name="text">
             <string>Crear Nueva Publicación</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollAreaPublicaciones">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>600</width>
               <height>511</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageSolicitudes">
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QGroupBox" name="groupBoxUsuarios">
          <property name="title">
           <string>Usuarios</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QTableWidget" name="tableWidgetUsuarios">
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Nombres</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Correo</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Nacimiento</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Acción</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QGroupBox" name="groupBoxSolicitudesRecibidas">
            <property name="title">
             <string>Solicitudes Recibidas</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <widget class="QTableWidget" name="tableWidgetSolicitudesRecibidas">
               <column>
                <property name="text">
                 <string>Correo</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Acciones</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBoxSolicitudesEnviadas">
            <property name="title">
             <string>Solicitudes Enviadas</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <item>
              <widget class="QTableWidget" name="tableWidgetSolicitudesEnviadas">
               <column>
                <property name="text">
                 <string>Correo</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Acciones</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageReportes">
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QGroupBox" name="groupBoxFechasMasPublicaciones">
            <property name="title">
             <string>Fechas con más publicaciones</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_14">
             <item>
              <widget class="QTableWidget" name="tableWidgetFechasMasPublicaciones">
               <column>
                <property name="text">
                 <string>Fecha</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Cantidad de publicaciones</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBoxPublicacionesMasComentarios">
            <property name="title">
             <string>Publicaciones con más comentarios</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_15">
             <item>
              <widget class="QTableWidget" name="tableWidgetPublicacionesMasComentarios">
               <column>
                <property name="text">
                 <string>Fecha</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Usuario</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>No. Comentarios</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QPushButton" name="pushButtonGenerarReportes">
          <property name="text">
           <string>Generar reportes</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_16">
            <item>
             <widget class="QGroupBox" name="groupBoxBST">
              <property name="title">
               <string>BST</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_17">
               <item>
                <widget class="QComboBox" name="comboBoxBST">
                 <item>
                  <property name="text">
                   <string>Todo</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item alignment="Qt::AlignmentFlag::AlignHCenter">
                <widget class="QPushButton" name="pushButtonGenerarBST">
                 <property name="text">
                  <string>Generar</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGraphicsView" name="graphicsViewBST"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pagePerfil">
       <layout class="QVBoxLayout" name="verticalLayout_18">
        <item>
         <widget class="QGroupBox" name="groupBoxPerfil">
          <property name="title">
           <string>Perfil</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_19">
           <item>
            <widget class="QLabel" name="labelNombres1">
             <property name="text">
              <string>Nombres</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditNombres"/>
           </item>
           <item>
            <widget class="QLabel" name="labelApellidos1">
             <property name="text">
              <string>Apellidos</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditApellidos"/>
           </item>
           <item>
            <widget class="QLabel" name="labelCorreo1">
             <property name="text">
              <string>Correo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditCorreo1"/>
           </item>
           <item>
            <widget class="QLabel" name="labelContrasena">
             <property name="text">
              <string>Contraseña</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditContrasena"/>
           </item>
           <item>
            <widget class="QLabel" name="labelFechaNacimiento1">
             <property name="text">
              <string>Fecha de nacimiento</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEditFechaNacimiento"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QPushButton" name="pushButtonModificarDatos">
               <property name="text">
                <string>Modificar datos</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonEliminarCuenta">
               <property name="text">
                <string>Eliminar cuenta</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1250</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBuscar">
    <property name="title">
     <string>Buscar</string>
    </property>
    <addaction name="actionBuscar"/>
   </widget>
   <widget class="QMenu" name="menuPublicaciones">
    <property name="title">
     <string>Publicaciones</string>
    </property>
    <addaction name="actionPublicaciones"/>
   </widget>
   <widget class="QMenu" name="menuSolicitudes">
    <property name="title">
     <string>Solicitudes</string>
    </property>
    <addaction name="actionSolicitudes"/>
   </widget>
   <widget class="QMenu" name="menuPerfil">
    <property name="title">
     <string>Perfil</string>
    </property>
    <addaction name="actionPerfil"/>
   </widget>
   <widget class="QMenu" name="menuReportes">
    <property name="title">
     <string>Reportes</string>
    </property>
    <addaction name="actionReportes"/>
   </widget>
   <addaction name="menuBuscar"/>
   <addaction name="menuPublicaciones"/>
   <addaction name="menuSolicitudes"/>
   <addaction name="menuReportes"/>
   <addaction name="menuPerfil"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBuscar">
   <property name="text">
    <string>Buscar</string>
   </property>
  </action>
  <action name="actionPublicaciones">
   <property name="text">
    <string>Publicaciones</string>
   </property>
  </action>
  <action name="actionSolicitudes">
   <property name="text">
    <string>Solicitudes</string>
   </property>
  </action>
  <action name="actionPerfil">
   <property name="text">
    <string>Perfil</string>
   </property>
  </action>
  <action name="actionReportes">
   <property name="text">
    <string>Reportes</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
